### ISSUE TITLE
NIH: de-duplicate items collecting funding rounds

### ISSUE THREAD

**Issue by mindrones** [nestauk/nesta#135]
As we found multiple items with different funding amounts, these are clearly different funding rounds for the same project.

To solve this we are discussing about aggregating rounds as a new field of the item we could call "earliest subproject", so that (pseudo code/names):

```js
[
  {id: 1, name: "foo", date_start: "2001-01-01", date_end: "2011-01-01", cost_ref: 100, year_fiscal_funding: 2010},
  {id: 2, name: "foo", date_start: "2003-03-03", date_end: "2013-03-03", cost_ref: 300, year_fiscal_funding: 2012},
  {id: 3, name: "foo", date_start: "2005-05-05", date_end: "2015-05-05", cost_ref: 500, year_fiscal_funding: 2014},
  {id: 4, name: "bar", date_start: "2002-02-02", date_end: "2004-04-04", cost_ref: 200, year_fiscal_funding: 2003},
]
```

would become:

```js
[
  {id: 1, name: "foo", date_start: "2001-01-01", date_end: "2015-05-05", cost_ref: 900, year_fiscal_funding: 2010,
    cost_rounds: [
      {id: 1, date_end: "2011-01-01", year: 2010, cost_ref: 100},
      {id: 2, date_end: "2013-03-03", year: 2012, cost_ref: 300},
      {id: 3, date_end: "2015-05-05", year: 2014, cost_ref: 500},
    ],
  },
  {id: 4, name: "bar", date_start: "2002-02-02", date_end: "2004-04-04", cost_ref: 200, year_fiscal_funding: 2003},
]
```

where:
- for aggregated items,
   - `date_start` is the `date_start` of the earliest subproject
   - `date_end` is the `date_end` of the latest subproject
   - `year_fiscal_funding` is the `year_fiscal_funding` of the earliest subproject
- suprojects have format `[{id, date_end, year, cost_ref}]`

Related: https://github.com/nestauk/nesta/pull/112#issuecomment-516368072

[1] See for example top two results at: https://search-health-scanner-5cs7g52446h7qscocqmiky5dn4.eu-west-2.es.amazonaws.com/nih_v5/_search?q=Motion%20Compensated%20Brain%20PET%20Imaging%20for%20Neuroscience%20Research&_source=title_of_project,date_end_project,date_start_project,year_fiscal_funding
<img width="653" alt="Screenshot 2019-07-30 at 12 09 04" src="https://user-images.githubusercontent.com/1309648/62126296-48bb9600-b2c7-11e9-988f-641259bfea3a.png">

- the second item is the first round, funded in 2015, project being expected to end in 2017
- the first item is the second round, funded in 2016, project being expected to end in 2019

**Comment by mindrones** [comment]
Note that [`booleanFlag_duplicate_abstract` will be deprecated](https://github.com/nestauk/nesta/pull/112#discussion_r312838187)

**Comment by mindrones** [comment]
Closed by #112
